recipes/qbs/all/conanfile.py

```diff
@@ -0,0 +1,98 @@
+from conans import ConanFile, tools
+from pathlib import Path
+
+class QbsConan(ConanFile):
+    name = "qbs"
+    description = "The Qbs build system"
+    topics = ("conan", "qbs", "build", "automation")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://qbs.io"
+    license = "LGPL-2.1-only", "LGPL-3.0-only", "Nokia-Qt-exception-1.1"
+    settings = {
+        "arch": ["x86_64"],
+        "os": [
+            "Linux",
+            "Macos",
+            "Windows"
+        ]
+    }
```

Comment:
```suggestion
    settings = "arch", "os"
```

It's better using configure + ConanInvalidConfiguration. Here, it will result in a ConanException and will break the CI job.

Reply:
Done